src/Admin/Pool.php

```diff
@@ -291,6 +292,27 @@ public function getAdminByAdminCode($adminCode)
         return $admin;
     }
 
+    /**
+     * See if an admin with a certain admin code exists.
+     *
+     * @param string $adminCode
+     *
+     * @return bool
+     */
+    final public function hasAdminByAdminCode(string $adminCode): bool
+    {
+        try {
+            if (false === $this->getAdminByAdminCode($adminCode)) {
```

Comment:
With this change you could remove the last `return true` statement.

```suggestion
            return $this->getAdminByAdminCode($adminCode) instanceof AdminInterface;
```

Reply:
I could do `instanceof AdminInterface` :+1: 

I myself always find it better for readability when you see that if this or that happens it returns false or when it passes all of that, in the end it will return true.